jitsi_meet/templates/etc/nginx/nginx.conf.j2

```diff
@@ -0,0 +1,69 @@
+# {{ ansible_managed }}
+
+user {{ jitsi_meet_nginx_user }};
+worker_processes {{ jitsi_meet_nginx_worker_processes }};
+pid /var/run/nginx.pid;
+
+events {
+    worker_connections {{ jitsi_meet_nginx_worker_connections }};
+    #multi_accept on;
+}
+
+http {
+    ##
+    # Basic Settings
+    ##
+
+    sendfile on;
+    tcp_nopush on;
+    tcp_nodelay on;
+    keepalive_timeout 65;
+    types_hash_max_size 2048;
+    server_tokens off;
+
+    #server_names_hash_bucket_size 64;
+    #server_name_in_redirect off;
+
+    include /etc/nginx/mime.types;
+    default_type application/octet-stream;
+
+    ##
+    # SSL Settings
+    ##
+
+    ssl_prefer_server_ciphers on;
+    ssl_protocols {{ jitsi_meet_nginx_ssl_protocols | join(' ') }};
+    ssl_ciphers {{ jitsi_meet_nginx_ssl_ciphers | join(':') }};
+
+    ssl_session_cache shared:SSL:50m;
+    ssl_session_timeout 5m;
+
+    ##
+    # Logging Settings
+    ##
+
+    access_log /var/log/nginx/access.log;
+    error_log /var/log/nginx/error.log;
+
+    ##
+    # Gzip Settings
+    ##
+
+    gzip                on;
+    gzip_disable        "msie6";
+
+    #gzip_vary          on;
+    #gzip_proxied       any;
+    gzip_comp_level     {{ jitsi_meet_nginx_gzip_comp_level }};
+    gzip_min_length     {{ jitsi_meet_nginx_gzip_min_length }};
+    #gzip_buffers        16 8k;
+    #gzip_http_version  1.1;
+    gzip_proxied        {{ jitsi_meet_nginx_gzip_proxied | join(" ") }};
+    gzip_types          {{ jitsi_meet_nginx_gzip_types | join(" ") }};
+
+    ##
+    # Virtual Host Configs
+    ##
+
+    include {{ jitsi_meet_nginx_vhosts_dir }}/*.conf;
+}
```

Comment:
Don't change this file, as this is managed by other an other role. If you really depend on nginx, then write a role dependency to the meta data and use this role.